test/integration/components/RecapComponent.js

```diff
@@ -0,0 +1,8 @@
+firstPersonBirthdate: 'individu-block:nth-of-type(1) strong',
+secondPersonBirthdate: 'individu-block:nth-of-type(2) strong',
+thirdPersonBirthdate: 'individu-block:nth-of-type(3) strong',
+fourthPersonBirthdate: 'individu-block:nth-of-type(4) strong',
```

Comment:
Not sure the little improvement of readability in the component is worth the extra file.

So I guess there is no way to have dynamic name for components elements, like :


```js
// Scenario
nthPersonBirthdate(1): DEMANDEUR_BIRTHDATE
nthPersonBirthdate(2): CONJOINT_BIRTHDATE
nthPersonBirthdate(3): CHILD_BIRTHDATE

// Component
nthPersonBirthdate: function(index) {
     return 'individu-block:nth-of-type(' + index + ') strong';
}
```